SingularityService/src/main/java/com/hubspot/singularity/mesos/SingularityMesosTaskBuilder.java

```diff
@@ -119,8 +119,9 @@ public SingularityTask buildTask(SingularityOfferHolder offerHolder, List<Resour
     } else if (MesosUtils.getDisk(offerHolder.getCurrentResources(), Optional.absent()) >= 1.0) {
       // If this offer contains 1MB of disk resources, claim it to enable disk usage reporting.
       // This is just a temporary hack to enable disk usage reporting where we can, not an actual way to match task disk requirements to offers.
```

Comment:
Maybe update this just to mention simply that disk usage won’t be reported if the disk resource isn’t > 0?